<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MarbleViewSettingsWidget</class>
 <widget class="QWidget" name="MarbleViewSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>541</width>
    <height>430</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_units">
     <property name="title">
      <string>&amp;Units</string>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_distanceUnit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>118</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>&amp;Distance:</string>
        </property>
        <property name="buddy">
         <cstring>kcfg_distanceUnit</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="kcfg_distanceUnit">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;The unit that gets used to measure altitude, lengths and distances (e.g. km, mi, ft).&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <item>
         <property name="text">
          <string>Kilometer, Meter</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Miles, Feet</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_angleUnit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>An&amp;gle:</string>
        </property>
        <property name="buddy">
         <cstring>kcfg_angleUnit</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="kcfg_angleUnit">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Specifies the notation of angles in coordinates: By default the Degree-Minute-Second notation (e.g. 54°30'00&quot; ) gets used. As an alternative you can choose decimal degrees (e.g. 54.5°).&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <item>
         <property name="text">
          <string>Degree (DMS)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Degree (Decimal)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Universal Transverse Mercator (UTM)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Military Grid Reference System (MGRS)</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_mapQuality">
     <property name="title">
      <string>Map &amp;Quality</string>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_stillQuality">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>118</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>&amp;Still image:</string>
        </property>
        <property name="buddy">
         <cstring>kcfg_stillQuality</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="kcfg_stillQuality">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Specifies the map quality that gets displayed while there is no user input. Usually this allows for high map quality as speed is no concern.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="currentIndex">
         <number>3</number>
        </property>
        <item>
         <property name="text">
          <string>Outline Quality</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Low Quality</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Normal</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>High Quality</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Print Quality</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_animationQuality">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>During &amp;animations:</string>
        </property>
        <property name="buddy">
         <cstring>kcfg_animationQuality</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="kcfg_animationQuality">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Specifies the map quality that gets displayed during map animations (e.g. while dragging the globe). &lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Especially on slow machines it is advisable to set this option to &quot;low quality&quot; as this will give better speed.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="currentIndex">
         <number>2</number>
        </property>
        <item>
         <property name="text">
          <string>Outline Quality</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Low Quality</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Normal</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>High Quality</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Print Quality</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_graphicsSystem">
        <property name="text">
         <string>G&amp;raphics system:</string>
        </property>
        <property name="buddy">
         <cstring>kcfg_graphicsSystem</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="kcfg_graphicsSystem">
        <property name="toolTip">
         <string>Specifies the graphics system that Qt uses to draw the map.
Changing this setting might be a good idea if there are performance issues with the system's graphics drivers.</string>
        </property>
        <item>
         <property name="text">
          <string>Native</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Raster (Software rendering)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>OpenGL</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>5</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_textLabels">
     <property name="title">
      <string>&amp;Text Labels</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QLabel" name="label_labelLocalization">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>118</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>&amp;Place names:</string>
          </property>
          <property name="buddy">
           <cstring>kcfg_labelLocalization</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="kcfg_labelLocalization">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Places often have different names in different languages. The label on the map can show the name in the user's language. Alternatively it can display the name in the official language and glyphs of that place. &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <item>
           <property name="text">
            <string>Custom &amp; Native Language</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Custom Language</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Native Language</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QLabel" name="label_mapFont">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>118</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Default map &amp;font:</string>
          </property>
          <property name="buddy">
           <cstring>kcfg_mapFont</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QFontComboBox" name="kcfg_mapFont">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;The default font that gets used on the map.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>446</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
